companies/zucchetti.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "zucchetti",
+    "relevant-countries": [
+        "it",
+        "us",
+        "spa",
+        "br",
+        "fr"
+    ],
+    "categories": [
+        "it",
+        "commerce"
+    ],
+    "name": "Zucchetti",
+    "address": "Via Solferino 1, 26900 Lodi - Italia",
+    "phone": "+39 0371 594-1",
```

Comment:
The privacy policy lists the phone number of the DPO. That should be used instead.